snap/snapcraft.yaml

```
@@ -126,6 +126,14 @@ adopt-info: openstack-exporter
 confinement: strict
 grade: stable
 
+platforms:
+  amd64:
+    build-on: [amd64]
+    build-for: [amd64]
+  arm64:
+    build-on: [arm64]
+    build-for: [arm64]
```

```suggestion
  amd64:
  arm64:
```

Suggesting this to make it easier to read. See https://snapcraft.io/docs/reference-architectures :

>  If the platform name is a valid debian architecture, then build-on and build-for can be omitted (see below for details).